daphne/src/lib.rs

```diff
@@ -225,6 +236,45 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_window_size`, `min_duration_start`, and `max_duration_end`
+//            should be reflected to the DAP spec.
+#[derive(Deserialize, Serialize)]
+#[serde(try_from = "ShadowDapGlobConfig")]
```

Comment:
This shadow struct isn't needed because we don't have any fields that require custom serialization.
```suggestion
```

Reply:
Completely forgot about this! Thanks for pointing it out.